packages/marko-web-gam/browser/inject-ads.vue

```diff
@@ -43,6 +43,14 @@ export default {
     const { childSelector } = this;
     const component = this;
     const $el = $(this.selector);
+
+    // Remove empty tags
+    $('p', $el).each(function removeEmptyTags() {
+      if ($(this).html().replace(/\s|&nbsp;/g, '').length === 0) {
```

Comment:
Wouldn't `$(this).text().trim().length` be better? Also, it will be considerably safer to not remove the element from the active DOM (which this will do) and instead skip the element.

I believe you could simply add a `trim` to line 58, and then add a return statement on the next line if `length === 0`

Reply:
This will also account for situations such as `<p><span></span></p>`

Reply:
Actually, the return statement isn't necessary: simply trimming the `.text()` value should suffice, as the character count will not be incremented (since the length will be `0`), thereby effectively "skipping" the empty element